### ISSUE TITLE
[7.0.7-35] test suite fails with libpng 1.6.31

### ISSUE THREAD

**Issue by remicollet** [ImageMagick/ImageMagick#1147]
```
============================================================================
Testsuite summary for ImageMagick 7.0.7
============================================================================
# TOTAL: 86
# PASS:  82
# SKIP:  0
# XFAIL: 0
# FAIL:  4
# XPASS: 0
# ERROR: 0
============================================================================
See ./test-suite.log
Please report to https://github.com/ImageMagick/ImageMagick/issues
============================================================================
```

In test-suite.log, I have tons of (despite the file is there)
`lt-validate: unable to load module '/builddir/build/BUILD/ImageMagick-7.0.7-35/coders/png.la': file not found @ error/module.c/OpenModule/1275.
`

**Comment by remicollet** [comment]
Also have a build warning
```
BUILDSTDERR: coders/png.c: In function 'read_eXIf_chunk':
BUILDSTDERR: coders/png.c:1955:7: warning: implicit declaration of function 'png_get_eXIf_1'; did you mean 'png_get_eXIf'? [-Wimplicit-function-declaration]
BUILDSTDERR:    if (png_get_eXIf_1(ping,info,&size,&data))
BUILDSTDERR:        ^~~~~~~~~~~~~~
BUILDSTDERR:        png_get_eXIf

```
The missing symbol probably explain why this coder cannot be loaded.

**Comment by remicollet** [comment]
From libpng 1.6.32 changelog:
` Changed name of png_get_eXIF and png_set_eXIf() to png_get_eXIf_1() and png_set_eXIf_1(), respectively, to avoid breaking API compatibility with libpng-1.6.31. `

**Comment by remicollet** [comment]
Notice: v6 is not affected.

**Comment by remicollet** [comment]
7.0.7-39 still affected

**Comment by remicollet** [comment]
Seems fixed in 7.0.8-0
